content/influxdb/cloud/account-management/switch-account.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Switch InfluxDB Cloud accounts
+seotitle: Switch between InfluxDB Cloud accounts
+description: >
+  Switch from one InfluxDB Cloud account to another and set a default account.  
+menu:
+  influxdb_cloud:
+    name: Switch InfluxDB accounts
+    parent: Account management
+weight: 105
+---
+If you belong to more than one {{< cloud-name >}} account with the same email address, you can switch from one account to another while staying logged in.
+
+You can also set a default account.
+
+To switch {{< cloud-name "short" >}} accounts:
+
+1. In the {{< cloud-name "short" >}} UI, select the **user avatar** in the left
+   navigation menu, and select **Account** > **Settings**.
+2. Click **Switch Account**.
```

Comment:
Let's also mention that there is no switch account option if a user is only associated with one account.